sip/sip-007-stacking-consensus.md

```diff
@@ -367,6 +367,70 @@ block attack --- not only must the attacker collude amongst a large
 fraction of mining power, but they must also collude amongst a
 majority of the Stacking participants in their block.
 
+# Stacker Delegation
+
+The process of delegation allows a Stacks wallet address (the
+represented address) to designate another address (the delegate
+address) for participating in the Stacking protocol. This delegate
+address, for as long as the delegation is valid, is able to sign and
+broadcast Stacking messages (i.e., messages which lock up Stacks,
+designate the Bitcoin reward address, and signal support for chain
+tips) on behalf of the represented address. This allows the owner of
+the represented address to contribute to the security of the network
+by having the delegate address signal support for chain tips. This
+combats potential attacks on the blockchain stability by miners that
+may attempt to mine hidden forks, hide eventually invalid forks, and
+other forms of miner misbehavior.
+
+Supporting delegation adds two new transaction types to the Stacks
+blockchain:
+
+* **Delegate Funds.** This transaction initiates a
+  represented-delegate relationship. It carries the following data:
+    * Delegate address
+    * End Block: the Bitcoin block height at which this relationship
+      terminates, unless a subsequent delegate funds transaction updates
+      the relationship.
+    * Delegated Amount: the total amount of Stacks from this address
```

Comment:
"Stacks" or "STX"?

Reply:
Aren't those equivalent?

Reply:
I don't think they are.  In my mind, "Stacks" is the protocol, and "STX" is the token (similar, "Bitcoin" is the protocol, and "BTC" is the token).  But, I don't have a strong opinion either way, just as long as we're consistent.

Reply:
Changed in c098f5d